src/app/app.component.ts

```diff
@@ -1,10 +1,33 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { ReactiveFormsModule, FormControl } from '@angular/forms';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+// import {MatButtonModule, MatFormFieldModule, MatInputModule, MatRippleModule} from '@angular/material'
```

Comment:
Please remove unused code prior to merging pull request.  If code should remain please add a comment describing why it should not be removed